EventListener/BodyListener.php

```diff
@@ -86,13 +86,13 @@ public function onKernelRequest(GetResponseEvent $event)
         $contentType = $request->headers->get('Content-Type');
         $normalizeRequest = $this->normalizeForms && $this->isFormRequest($request);
 
-        if ($this->isDecodeable($request)) {
-            $format = null === $contentType
-                ? $request->getRequestFormat()
-                : $request->getFormat($contentType);
-
-            $format = $format ?: $this->defaultFormat;
+        if (null === $contentType) {
+            $format = $request->getRequestFormat($this->defaultFormat);
```

Comment:
I understood, but `$this->isDecodeable` doesn't supports "GET" method, causing error on deserialization.